components/ResultsPage/Modal.tsx

```diff
@@ -0,0 +1,60 @@
+import { getTranslations } from '../../i18n/api'
+import { WebTranslations } from '../../i18n/web'
+import { MaritalStatus } from '../../utils/api/definitions/enums'
+import { useTranslation } from '../Hooks'
+
+export const Modal: React.VFC<{
+  isOpen
+  onClose
+  position
+  partner
+  maritalStatus
+}> = ({ isOpen, onClose, position, partner, maritalStatus }) => {
```

Comment:
I don't see where the "position" prop is being used. Probably don't need it, can remove from state in parent component and just center it on the screen (as is happening now).